clcache.py

```diff
@@ -850,7 +925,7 @@ def analyze(cmdline):
             raise MultipleSourceFilesComplexError()
 
         if len(inputFiles) == 1:
-            if 'Fo' in options:
+            if 'Fo' in options and len(options['Fo']) > 0:
```

Comment:
[PEP 8](https://www.python.org/dev/peps/pep-0008/) explains:

> For sequences, (strings, lists, tuples), use the fact that empty sequences are false.
> 
> Yes: if not seq:
>     if seq:
>
> No: if len(seq):
>    if not len(seq):

Hence, I'd say `if 'Fo' in options and options['Fo']:` would be more idiomatic.